bom/pom.xml

```diff
@@ -166,6 +181,56 @@
          </dependency>
          <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-distexec-demo</artifactId>
+            <version>${project.version}</version>
+         </dependency>
+         <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-gridfs-webdav</artifactId>
+            <version>${project.version}</version>
```

Comment:
Actually Martin has a point here. With our [new SNAPSHOT deployment](https://repository.jboss.org/nexus/content/repositories/snapshots/org/infinispan/infinispan-bom/9.0.0-SNAPSHOT) (which deploys new SNAPSHOTs whenever we have a stable build) it is perfectly legal to build demos without other modules (of course whether it's sensible or not is another story...)